### DISCUSSION TITLE
how to implement the service in google cloud

### DISCUSSION THREAD

**Discussion by Rotman-IS** [saran-surya/email_auth_node#40] [Q&A]
hello I would like to know how to implement the email_auth service in google cloud?

**Comment by saran-surya** [comment]
If you are using gcp, they provide an inbuilt solution for it I believe,  I will check and update you regarding it

**Reply by saran-surya** [reply]
> hello I would like to know how to implement the email_auth service in google cloud?

Hi are you till facing any issues regarding it ?
